main.cpp

```diff
@@ -194,19 +241,24 @@ struct VaccumCleaner
     std::string informUserWhenFull();
 };
 
-/*
-Thing 3) Bicycle
-5 properties:
-    1) number of gears (int)
-    2) type (std::string)
-    3) speed (int)
-    4) cadence (std::string)
-    5) color (std::string)
-3 things it can do:
-    1) cycle at a certain speed
-    2) turn left or right
-    3) break
- */
+void VaccumCleaner::dustCollect(int dustQuantity, int time)
+{
+    if (dustQuantity != 0) {
+        dustQuantity -= 1;
+        time -= 1;
+    }
```

Comment:
curly braces go on their own line
conform to the course coding standard
- [ ] revise